src/dbSta/src/dbSta.cc

```diff
@@ -270,6 +271,129 @@ std::set<dbNet*> dbSta::findClkNets(const Clock* clk)
   return clk_nets;
 }
 
+std::map<dbSta::InstType, int> dbSta::getInstancesType()
```

Comment:
I suggest change the name of this function countInstancesByType. It's more descriptive of it's actual functionality.

Reply:
I agree, this is a better name.